src/app/workspace/(crew-work)/crew/page.tsx

```diff
@@ -16,7 +16,7 @@ const Crew = async ({
 
     const initialState: PageState = initPageState();
 
-    const { data }: { data: Work | null } = await getWorkWithCrewMetaData({
+    const { data }: { data: Work | null } = await getCrewForCrewHomepage({
```

Comment:
```suggestion
    const { data }: { data: Work | null } = await getWorkForCrewHomePage({
```

inconsistency issue, naming and code styling should be consistent